### PR TITLE
Update icon_analysis.py

### PR DESCRIPTION
**Pull request by superpoussin22**
manage the case were icon name is None

### What was a problem?

```
In some case icon_name si not defined need to handle this case 
```

### How this PR fixes the problem?

```
Add a test on icon_name is None and handle the case
```

### Check lists (check `x` in `[ ]` of list items)

- [X] Run MobSF unit tests
- [X ] Tested Working on Linux, Mac
- [X] Coding style (indentation, etc)

### Additional Comments (if any)

```

```

### CODE REVIEW

File: StaticAnalyzer/views/android/icon_analysis.py [hunk 1]
```diff
@@ -112,6 +112,11 @@ def get_icon(apk_path, res_dir):
         a = apk.APK(apk_path)
         icon_resolution = 0xFFFE - 1
         icon_name = a.get_app_icon(max_dpi=icon_resolution)
+        if icon_name is None:
+            return {
+            'path': guess_icon_path(res_dir),
+            'hidden': True
+        }
```
Comment (ajinabraham) [inline StaticAnalyzer/views/android/icon_analysis.py]: Is this indentation correct @superpoussin22 ?
Did you run unit test after making the changes?

File: StaticAnalyzer/views/android/icon_analysis.py [hunk 2]
```diff
@@ -112,6 +112,11 @@ def get_icon(apk_path, res_dir):
         a = apk.APK(apk_path)
         icon_resolution = 0xFFFE - 1
         icon_name = a.get_app_icon(max_dpi=icon_resolution)
+        if icon_name is None:
```
[the remark this answers is not in the document]
Comment (superpoussin22) [reply StaticAnalyzer/views/android/icon_analysis.py]: yes it is

File: StaticAnalyzer/views/android/icon_analysis.py (hunk 1 above)
Comment (superpoussin22) [reply StaticAnalyzer/views/android/icon_analysis.py]: bad copy paste which break ident , solved

**Comment by ajinabraham** [comment]
@superpoussin22 Good to merge this!
